packages/component-library/src/components/form/_internal/mt-base-field/mt-base-field.vue

```diff
@@ -261,6 +261,7 @@ $mt-field-transition:
   .mt-field__help-text {
     margin-left: var(--scale-size-4);
     align-self: center;
+    z-index: 2100;
```

Comment:
Shouldn't the inline style in `mt-tooltip` be enough? Or am I missing something here?

And please add a changelog to this PR. The rest looks good.

Reply:
Done 👍